repository/full/org.eclipse.rcptt.product

```diff
@@ -298,14 +298,12 @@
       <feature id="org.eclipse.emf.databinding"/>
       <feature id="org.eclipse.rcptt.module.nebula"/>
       <feature id="org.eclipse.rcptt.module.testrail"/>
-      <feature id="org.eclipse.rcptt.module.zephyr"/>
```

Comment:
Why is this removed?

Reply:
Btw. you don't have to explicitly add all dependencies to the product definition.
It is sufficient to just list the root plugins/features you actually want and let PDE/Tycho include all transitive dependencies when launching a product from the IDE or building it.

Reply:
In this case, there is no implicit dependency in Zephyr, so the line is necessary. The approach is historical - PDE failed to take into account feature dependencies in early 201x, so plugin configuration mode as used with all plugins explicitly listed. It should be possible to use only root feature as the bug is  fixed, but this task would require a significant work, to build a new feature dependency graph.